api/tests/tests.py

```diff
@@ -69,22 +69,22 @@ def setUp(self):
         )
 
 
-class LoginViewTest(BaseTest):
-    def test_login_instructor(self):
-        url = reverse("login")
-        data = {"username": "test@gmail.com", "password": "password"}
-        response = self.client.post(url, data, format="json")
-        response_data = response.json()
-        self.assertEqual(response.status_code, 200)
-        self.assertTrue("instructor" in response_data)
-
-    def test_login_student(self):
-        url = reverse("login")
-        data = {"username": "student@gmail.com", "password": "password"}
-        response = self.client.post(url, data, format="json")
-        response_data = response.json()
-        self.assertEqual(response.status_code, 200)
-        self.assertTrue("student" in response_data)
+# class LoginViewTest(BaseTest):
+#     def test_login_instructor(self):
+#         url = reverse("login")
+#         data = {"username": "test@gmail.com", "password": "password"}
+#         response = self.client.post(url, data, format="json")
+#         response_data = response.json()
+#         self.assertEqual(response.status_code, 200)
+#         self.assertTrue("instructor" in response_data)
+
+#     def test_login_student(self):
+#         url = reverse("login")
+#         data = {"username": "student@gmail.com", "password": "password"}
+#         response = self.client.post(url, data, format="json")
+#         response_data = response.json()
+#         self.assertEqual(response.status_code, 200)
+#         self.assertTrue("student" in response_data)
```

Comment:
Was there a reason why you didn't remove this, @aprask?

Reply:
I'll go ahead and remove this, I missed this.